example/CMakeLists.txt

```diff
@@ -47,18 +53,8 @@ ExternalProject_Add(
     CMAKE_CACHE_ARGS "-D CUSTOM_PROXY_FACTORY_INCLUDE_DIR:STRING=${CUSTOM_PROXY_FACTORY_INCLUDE_DIR}"
                      "-D CUSTOM_PROXY_FACTORY_SOURCES:STRING=${CUSTOM_PROXY_FACTORY_SOURCES}"
                      "-D CUSTOM_PROXY_SOURCES:STRING=${CUSTOM_PROXY_SOURCES}"
-                     "-D CMAKE_CXX_COMPILER_LAUNCHER:STRING=ccache"
-    # TODO: What should the install step do?
-    INSTALL_COMMAND ""
+    # Execute LIBMEXCLASS_INSTALL_STEPS_FILE as a script.    
+    INSTALL_COMMAND ${CMAKE_COMMAND} -D CMAKE_INSTALL_DIR=${CMAKE_INSTALL_DIR} -P ${LIBMEXCLASS_INSTALL_STEPS_FILE}
 )
 
-# Get the source directory for libmexclass.
-ExternalProject_Get_Property(${LIBMEXCLASS_EXTERNAL_PROJECT_TARGET_NAME} SOURCE_DIR)
-# Get the instllation directory for libmexclass.
-ExternalProject_Get_Property(${LIBMEXCLASS_EXTERNAL_PROJECT_TARGET_NAME} BINARY_DIR)
-
-set(LIBMEXCLASS_MATLAB_SOURCES ${SOURCE_DIR}/../matlab/)
-
-# TODO: Only copy libmexclass.so and gateway.mexa64
-install(DIRECTORY ${BINARY_DIR}/ DESTINATION libmexclass)
-install(DIRECTORY ${LIBMEXCLASS_MATLAB_SOURCES} DESTINATION libmexclass)
+install(CODE "MESSAGE(\"Completed install step for ${CLIENT_PROJECT_NAME}.\")")
```

Comment:
I think we can probably remove this line unless we feel that it isn't clear enough that the external project build has completed successfully.

Reply:
Fixed in commit 309a468 by updating the client install step to take care of copying the packaged libmexclass install to the final destination.